package.json

```diff
@@ -35,23 +35,23 @@
     "date-fns": "^3.3.1",
     "date-fns-tz": "^3.1.3",
     "iso-639-3": "^3.0.1",
-    "isomorphic-dompurify": "^2.14.0",
     "jwt-decode": "^4.0.0",
-    "marked": "^14.0.0",
     "next": "14.2.7",
     "next-auth": "^4.24.5",
     "next-runtime-env": "^3.2.0",
     "react": "^18",
     "react-dom": "^18",
+    "react-markdown": "^9.0.1",
     "react-phone-number-input": "^3.4.3",
+    "rehype-slug": "^6.0.0",
+    "remark-gfm": "^4.0.0",
     "sharp": "^0.33.2",
     "tailwind-merge": "^2.4.0",
     "tailwindcss-animate": "^1.0.7",
     "utils": "^0.2.2"
   },
   "devDependencies": {
     "@types/axios": "^0.14.0",
-    "@types/dompurify": "^2.3.0",
```

Comment:
I suggest keeping `dompurify` or use `rehype-sanitize` as markdown content is coming from user-created sources.